canonical/ubuntu.com
# chore: Update name of  /security/compliance-automation and add a redirect

## Done

- Updates based on [copydoc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit) changes from Yash Lad
- Updates URL to `/security/compliance-automation`
- Adds a redirect

## QA

- Go to https://ubuntu-com-14176.demos.haus/security/compliance-automation and check the content matches the [copydoc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit)
- Check the URL change matches the [copydoc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit) name
- Check the redirects work
- Finally resolve the associated comments

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14153
